config/testgrids/openshift/openshift.yaml

```diff
@@ -96,117 +96,621 @@ dashboards:
     description: Runs Kubernetes e2e tests against an OpenShift 4.1 cluster.
     test_group_name: release-openshift-origin-installer-e2e-aws-4.1
     base_options: width=10
+    open_test_template: # The URL template to visit after clicking on a cell
+      url: https://prow.svc.ci.openshift.org/view/gcs/<gcs_prefix>/<changelist>
+    file_bug_template: # The URL template to visit when filing a bug
+      url: https://github.com/openshift/origin/issues/new
+      options:
+        - key: title
+          value: 'E2E: <test-name>'
+        - key: body
+          value: <test-url>
+    open_bug_template: # The URL template to visit when visiting an associated bug
+      url: https://github.com/openshift/origin/issues/
+    results_text: See these results on Prow # Text to show in the about menu as a link to another view of the results
```

Comment:
Shouldn't need to specify this if you don't want, this is the same as the current default